tests/lib/TestUtils.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * TestUtils.cpp
+ * Functions used for unit testing.
+ * Copyright (C) 2012 Simon Newton
+ */
+
+#include "TestUtils.h"
+
+#include <cppunit/extensions/HelperMacros.h>
+#include <string.h>
+#include <iostream>
+
+namespace testing {
+
+void _AssertDataEquals(
```

Comment:
The 'Private' is just a convention to mean 'don't depend on this'. The definitions live in the file with the base name.